pages/disability/how-to-file-claim.md

```diff
@@ -55,13 +55,19 @@ Find out how to file a claim for disability compensation.
 <h4 itemprop="name">What information do I need to file my claim?</h4>
 <div itemprop="itemListElement">
 
+For your first disability claim, we'll review your:
+
+- Discharge papers (DD214 or other separation documents)
+- Service treatment records <br>
+[(Order service records through the National Archives)](https://www.archives.gov/veterans/military-service-records)
+
 For all disability claims you'll have a chance to provide evidence to support your claim. Evidence could include:
```

Comment:
@bethpotts Did I resolve this line correctly? Or should it be `For all disability claims, we'll review your:`?